meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy

```diff
@@ -449,45 +449,88 @@ install() {
 update_ec() {
   verify_artifacts ec
   echo "Updating EC..."
-  dasharo_ectool flash "$EC_UPDATE_FILE" &>> $ERR_LOG_FILE
+  $DASHARO_ECTOOL flash "$EC_UPDATE_FILE" &>> $ERR_LOG_FILE
   error_check "Failed to update EC firmware"
 }
 
 update() {
-  local _update_version=""
+  local _can_switch_to_heads="false"
 
   check_if_ac
   error_check "Firmware update process interrupted on user request."
 
   echo "Checking for the latest Dasharo update available..."
   echo "Current Dasharo version: $DASHARO_VERSION"
-  if [ -v BIOS_HASH_LINK_DES ]; then
-    if [ -v DES_IS_LOGGED ]; then
-      echo "Latest available Dasharo version: $DASHARO_REL_VER_DES"
-      BIOS_HASH_LINK=$BIOS_HASH_LINK_DES
-      BIOS_SIGN_LINK=$BIOS_SIGN_LINK_DES
-      BIOS_LINK=$BIOS_LINK_DES
-      if [ "$HAVE_EC" = "true" ]; then
-        EC_HASH_LINK=$EC_HASH_LINK_DES
-        EC_SIGN_LINK=$EC_SIGN_LINK_DES
-        EC_LINK=$EC_LINK_DES
+  if [ -v DES_IS_LOGGED ]; then
```

Comment:
@miczyg1 did you test that switching on any real hardware and has logs from it?

Reply:
Not yet, only the script logic has been tested. The "flashed" binaries are assumed to be functional.

Reply:
Yes, I uploaded some logs here: https://github.com/Dasharo/meta-dts/pull/97#issuecomment-1960990490